tools/cli/adminCommands.go

```diff
@@ -263,7 +263,7 @@ func readOneRow(query *gocql.Query) (map[string]interface{}, error) {
 }
 
 func connectToCassandra(c *cli.Context) *gocql.Session {
-	host := getRequiredOption(c, FlagAddress)
+	host := getRequiredOption(c, FlagCassandraAddress)
```

Comment:
could you also add a `FlagCassandraPort`?

Reply:
line 270 still use the `FlagPort `

Reply:
done